uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
```

Comment:
for the sake of getting this PR (THE PR WHICH NEVER ENDS) out the door, I'm inclined to leave this, but I will leave a comment in the code that it can probably be removed.